src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F

```diff
@@ -347,16 +345,16 @@ subroutine driver_cloudiness(itimestep,configs,mesh,diag_physics,sfc_input,its,i
 
     case default
 
-    if (convection_scheme=="cu_grell_freitas") then
-       call mpas_timer_start('cal_gfcld')
-       call cal_gfcld(cldfrac_p , qcrad_p   , qirad_p    , &
-                      qccu_p    , qicu_p    ,              &
-                      its,  ite , jts , jte , kts , kte  )
-      call mpas_timer_stop('cal_gfcld')
-    endif
-
  end select cld_fraction_select
 
+ if (convection_scheme=="cu_grell_freitas") then
```

Comment:
similar to above, is the intention to always overwrite the fields calculated in the case statement above?

Reply:
I looked at cal_gfcld and understand now what is happening.